lib/gitWorkflow.js

```diff
@@ -65,11 +66,13 @@ const handleError = (error, ctx, symbol) => {
 }
 
 export class GitWorkflow {
-  constructor({ allowEmpty, gitConfigDir, gitDir, matchedFileChunks }) {
+  constructor({ allowEmpty, gitConfigDir, gitDir, matchedFileChunks, diff, diffFilter }) {
     this.execGit = (args, options = {}) => execGit(args, { ...options, cwd: gitDir })
     this.deletedFiles = []
     this.gitConfigDir = gitConfigDir
     this.gitDir = gitDir
+    this.diff = diff
+    this.diffFilter = diffFilter
     this.unstagedDiff = null
```

Comment:
Maybe this can be removed? It's not used anywhere in a lib

Reply:
Sure, go ahead. It's probably a leftover. 👍